skill/scenes.py

```diff
@@ -116,6 +117,155 @@ def handle_local_intents(self, request: Request):
         return ChooseScenario()
 
 
+# endregion
+
+# region settings
+
+
+class FirstSettingsScene(GlobalScene):
+    def reply(self, request: Request):
+        text, tts = texts.start_setting()
+        return self.make_response(request, text, tts, buttons=[button("Помощь")])
+
+    def handle_local_intents(self, request: Request):
+        if intents.FIO in request.entities_list:
+            return Settings_GetSchool()
+
+
+class Settings_GetSchool(GlobalScene):
+    def reply(self, request: Request):
+        name = request.entity(intents.FIO)[0]["first_name"].capitalize()
+
+        text, tts = texts.what_school(name)
+        return self.make_response(
+            request,
+            text,
+            tts,
+            buttons=[button("Помощь")],
+            state={state.TEMP_NAME: name},
+        )
+
+    def handle_local_intents(self, request: Request):
+        if intents.NUMBER in request.entities_list:
+            return Settings_GetClassNumber()
+
+
+class Settings_GetClassNumber(GlobalScene):
+    def reply(self, request: Request):
+        school_num = str(request.entity(intents.NUMBER)[0])
+        text, tts = texts.what_classnumber()
+        return self.make_response(
+            request,
+            text,
+            tts,
+            buttons=[button("Помощь")],
+            state={state.TEMP_SCHOOL: str(school_num)},
+        )
+
+    def handle_local_intents(self, request: Request):
+        if intents.NUMBER in request.entities_list:
+            class_num = request.entity(intents.NUMBER)[0]
+            if 1 <= class_num <= 11:
+                class_letter = request.tokens[-1]
+                if "А" <= class_letter.capitalize() <= "Я":
```

Comment:
```suggestion
                if "А" <= class_letter.upper() <= "Я":
```
Давай upper(), capitalize() как то непривычно смотрится - оно обычно, чтобы слово с большой буквы сделать